discovery-provider/tests/test_undisbursed_challeges.py

```diff
@@ -0,0 +1,154 @@
+from src.models import Challenge, UserChallenge, ChallengeType
+from src.utils.db_session import get_db
+from src.queries.get_undisbursed_challenges import get_undisbursed_challenges
+from tests.utils import populate_mock_db_blocks, clean_up_db
+
+
+def setup_challenges(app):
+    with app.app_context():
+        db = get_db()
+        clean_up_db(db)
```

Comment:
we should use text fixtures for this stuff https://stackoverflow.com/a/62784688/11435157